File: pset6/mario/less/mario.py
from cs50 import get_int

height = get_int("Height: ")
# check that if value of height entered is from 1 to 8
while True:
    if height < 1 or height > 8:
        height = get_int("Height: ")
    else:
        break

# prints a pyramid of given height
for i in range(1, height + 1):
    print(" " * (height - i), end="")
    print("#" * (i))

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n = 0;  // declare a variable n
    // get value of Height from user between 1 to 8
    do
    {
        n = get_int("Height : ");
    }
    while (n < 1 || n > 8);

    //Print pyramid
    for (int i = 1; i <= n; i++)
    {
        for (int j = n - i; j >= 1; j--)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }

    return 0;
}